main.go

```diff
@@ -190,18 +191,38 @@ func (o *Options) runCLIReportMode() error {
 		RawJobResultsAnalysisConfig: o.toRawJobResultsAnalysisConfig(),
 		DisplayDataConfig:           o.toDisplayDataConfig(),
 	}
-	var bugCache buganalysis.BugCache
-	if o.SkipBugLookup {
-		bugCache = buganalysis.NewNoOpBugCache()
-	} else {
-		bugCache = buganalysis.NewBugCache()
-	}
-	testReport := analyzer.PrepareTestReport(o.ToTestGridDashboardCoordinates()[0], bugCache)
+
+	testReport := analyzer.PrepareTestReport(o.ToTestGridDashboardCoordinates()[0], o.getVariantManager(), o.getBugCache())
 	enc := json.NewEncoder(os.Stdout)
 	enc.Encode(testReport)
 	return nil
 }
 
+func (o *Options) hasOpenshiftRelease() bool {
+	for _, dashboardCoordinate := range o.ToTestGridDashboardCoordinates() {
+		if len(dashboardCoordinate.OpenshiftRelease) > 0 {
+			return true
+		}
+	}
+	return false
+}
+
+func (o *Options) getBugCache() buganalysis.BugCache {
+	if o.SkipBugLookup || !o.hasOpenshiftRelease() {
+		return buganalysis.NewNoOpBugCache()
+	} else {
+		return buganalysis.NewBugCache()
+	}
+}
+
+func (o *Options) getVariantManager() testidentification.VariantManager {
+	if o.hasOpenshiftRelease() {
+		return testidentification.NewOpenshiftVariantManager()
```

Comment:
it's not immediately obvious to me why the set of variants should be tied to whether or not what we are analyzing includes an openshift release version?

Reply:
> it's not immediately obvious to me why the set of variants should be tied to whether or not what we are analyzing includes an openshift release version?

Depending on what test-grid you're looking at, you have different variants.  See https://testgrid.k8s.io/sig-release-master-informing for instance.  It's likely to have a kube set

Reply:
yeah, i get that you might have a reason to want different variants depending what set of testgrid jobs you are analyzing.  i'm just not convinced "we have an openshift release" is the right discriminator for choosing the appropriate set of variants.

Reply:
> yeah, i get that you might have a reason to want different variants depending what set of testgrid jobs you are analyzing. i'm just not convinced "we have an openshift release" is the right discriminator for choosing the appropriate set of variants.

So this is trying to determine: "am I looking at a set of testgrids that includes an openshift".  How would you like to determine that?